WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -363,13 +355,16 @@ public async Task RemoveAndStopAllAsync(CancellationToken cancel)
 
 					try
 					{
-						var keyManager = wallet.KeyManager;
-						string backupWalletFilePath = WalletDirectories.GetWalletFilePaths(Path.GetFileName(keyManager.FilePath)).walletBackupFilePath;
-						keyManager.ToFile(backupWalletFilePath);
-						Logger.LogInfo($"{nameof(wallet.KeyManager)} backup saved to `{backupWalletFilePath}`.");
-						await wallet.StopAsync(cancel).ConfigureAwait(false);
+						if (wallet.State >= WalletState.Initialized)
+						{
+							var keyManager = wallet.KeyManager;
+							string backupWalletFilePath = WalletDirectories.GetWalletFilePaths(Path.GetFileName(keyManager.FilePath)).walletBackupFilePath;
+							keyManager.ToFile(backupWalletFilePath);
+							Logger.LogInfo($"{nameof(wallet.KeyManager)} backup saved to `{backupWalletFilePath}`.");
```

Comment:
I would wrap this around a `try-catch` because IO operations can fail and in case we fail to backup one wallet then none will be backuped.

Reply:
There is a try-catch there - it will continue on other wallets.

Reply:
Ops sorry, I didn't see it.